cronos-play/getting-started_unreal/quick-start/get-tokens-or-transactions.md

```diff
@@ -0,0 +1,78 @@
+# Get Tokens or Transactions
+
+## GetTokensBlocking
+
+given the BlockScout REST API base url and the account address (hexadecimal), it will return the list of all owned tokens (ref: https://cronos.org/explorer/testnet3/api-docs)
+
+<figure><img src="../../../.gitbook/assets/image (19) (2).png" alt=""><figcaption></figcaption></figure>
+
+* `Blockscout Base Url`: the base url of the BlockScout API
```

Comment:
I'm wonder if https://cronos.org/explorer/testnet3/api is the correct API for BlockScout? If it's, then I think we should put within the docs @damoncro

Reply:
Done